Documentation/ApiOverview/Backend/LoginProvider.rst

```diff
@@ -60,17 +60,50 @@ replace its registered :php:`provider` class with your custom class.
 LoginProviderInterface
 ======================
 
-The LoginProviderInterface contains only one method:
+..  deprecated:: 13.3
+    Method :php:`LoginProviderInterface->render()` has been marked as deprecated
+    and is substituted by  :php:`LoginProviderInterface->modifyView()` that will
+    be added to the interface in TYPO3 v14, removing :php:`render()` from the
+    interface in v14. See section :ref:`login-provider-interface-migration`.
+
+The :php-short:`TYPO3\CMS\Backend\LoginProvider\LoginProviderInterface` contains
+only the deprecated `render()` method in TYPO3 v13.
 
 ..  include:: /CodeSnippets/Backend/LoginProviderInterface.rst.txt
 
+..  _login-provider-interface-migration:
+
+Migration
+---------
+
+Consumers of :php-short:`\TYPO3\CMS\Backend\LoginProvider\LoginProviderInterface`
+should implement :php:`modifyView()` instead of the :php:`render()` method.
```

Comment:
You write "instead of render()", but render() is currently part of the interface. So you have to implement it, or am I wrong.

So maybe: "... should implement modifyView() together with the render() method." or similar.

Reply:
It was actually written like that in the changelog, but I agree it sounds confusing.   Changed it